figures/models.py

```diff
@@ -116,6 +116,8 @@ class SiteDailyMetrics(TimeStampedModel):
     # Should change this to default value of 0
     mau = models.IntegerField(blank=True, null=True)
 
+    # TODO: Add field for number of CDMs reported
```

Comment:
**Nit:** This doesn't seem to be related to the pull request overall. Was it added by mistake?

Reply:
Nope, on purpose, a reminder while I was in the code